ipwb/util.py

```diff
@@ -39,16 +43,26 @@
 dtPattern = re.compile(r"^(\d{4})(\d{2})?(\d{2})?(\d{2})?(\d{2})?(\d{2})?$")
 
 
-def isDaemonAlive(hostAndPort="{0}:{1}".format(IPFSAPI_HOST, IPFSAPI_PORT)):
+def createIPFSClient():
```

Comment:
```suggestion
def createIPFSClient(daemonMultiaddr=IPFSAPI_MUTLIADDRESS):
```

Following the pattern used in `isDaemonAlive()`, it perhaps is better to also make it parametrized with the default value in place.

Reply:
1d57554